src/ApiService/ApiService/TimerDaily.cs

```diff
@@ -7,30 +7,26 @@ namespace Microsoft.OneFuzz.Service;
 public class TimerDaily {
     private readonly ILogger _logger;
 
-    private readonly IScalesetOperations _scalesets;
+    private readonly IOnefuzzContext _context;
 
-    private readonly IWebhookMessageLogOperations _webhookMessageLogs;
-
-    public TimerDaily(ILoggerFactory loggerFactory, IScalesetOperations scalesets, IWebhookMessageLogOperations webhookMessageLogs) {
+    public TimerDaily(ILoggerFactory loggerFactory, IOnefuzzContext context) {
         _logger = loggerFactory.CreateLogger<QueueTaskHearbeat>();
-        _scalesets = scalesets;
-        _webhookMessageLogs = webhookMessageLogs;
+        _context = context;
     }
 
-    //[Function("TimerDaily")]
+    [Function("TimerDaily")]
     public async Async.Task Run([TimerTrigger("1.00:00:00")] TimerInfo myTimer) {
-        var scalesets = _scalesets.Search();
+        var scalesets = _context.ScalesetOperations.Search();
         await foreach (var scaleset in scalesets) {
             _logger.LogInformation("updating scaleset configs: {ScaleSetId}", scaleset.ScalesetId);
-            // todo: do it in batches
-            await _scalesets.Replace(scaleset with { NeedsConfigUpdate = true });
+            await _context.ScalesetOperations.Replace(scaleset with { NeedsConfigUpdate = true });
```

Comment:
we should check if `scaleset.NeedsConfigUpdate` is already true and skip it, to avoid needless writes